frontend/src/components/form/base/EDatePicker.vue

```diff
@@ -55,30 +55,22 @@ export default {
   methods: {
     format (val) {
       if (val !== '') {
-        return this.$moment.utc(val, this.valueFormat, this.$i18n.locale).format('L')
+        return this.$date.utc(val, this.valueFormat).format('L')
       }
       return ''
     },
     formatPicker (val) {
       if (val !== '') {
-        return this.$moment.utc(val, this.valueFormat).format(this.$moment.HTML5_FMT.DATE)
+        return this.$date.utc(val, this.valueFormat).format('YYYY-MM-DD')
       }
       return ''
     },
     parse (val) {
       if (val) {
-        const m = this.$moment.utc(val, 'L', this.$i18n.locale)
-        if (m.isValid()) {
-          return Promise.resolve(m.format(this.valueFormat))
+        const parsedDate = this.$date.utc(val, 'L')
+        if (parsedDate.isValid() && parsedDate.format('L') === val) {
```

Comment:
Dayjs only checks if the date can be parsed successfully, but not for the validity of the date (i.e. February 31 or month number 13).